src/Components/SingleProductSlider/SingleProductSlider.js

```diff
@@ -0,0 +1,36 @@
+import { Swiper, SwiperSlide } from 'swiper/react';
+import SwiperCore, { Navigation, Thumbs } from 'swiper';
+import PropTypes from 'prop-types';
+import 'swiper/swiper-bundle.min.css';
+import './styles.scss';
+
+SwiperCore.use([Navigation, Thumbs]);
+
+const SingleProductSlider = ({ images }) => {
+  let slides = [];
+  images.map((image, index) =>
+    slides.push(
+      <SwiperSlide key={index}>
+        <img src={image.image.url} />
+      </SwiperSlide>
+    )
+  );
+
+  return (
+    <Swiper
+      loop={true}
+      spaceBetween={10}
+      navigation={true}
+      slidesPerView={1}
+      className="main-slider"
+    >
+      {slides}
+    </Swiper>
+  );
```

Comment:
there no need to create a variable to store components, variables usually store values used for the logic, not components

Also, `map` returns an array, when using `map` you don't need to create an empty array and then push items to that array, [in the documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) you can find examples of how to use `map`

```jsx
return (
  <Swiper
    loop={true}
    spaceBetween={10}
    navigation={true}
    slidesPerView={1}
    className="main-slider"
  >
    {images.map((image, index) => (
      <SwiperSlide key={index}>
        <img src={image.image.url} />
      </SwiperSlide>
    ))}
  </Swiper>
);
```